### ISSUE TITLE
FFS Rejection

### ISSUE THREAD

**Issue by 0xLeif** [zmeriksen/NoteMapV2#26]
# 1st Rejection
> Feb 18, 2018 at 1:18 PM
> From Apple
> 2. 1 Performance: App Completeness
> 4. 2 Design: Minimum Functionality
> Guideline 2.1 - Information Needed
> 
> 
> This type of app has been identified as one that may violate one or more of the following App Store Review Guidelines. Specifically, these types of apps often:
> 
> 1.1.6 - Include false information, features, or misleading metadata.
> 2.3.0 - Undergo significant concept changes after approval
> 2.3.1 - Have hidden or undocumented features, including hidden "switches" that redirect to a gambling or lottery website
> 3.1.1 - Use payment mechanisms other than in-app purchase to unlock features or functionality in the app
> 4.3.0 - Are a duplicate of another app or are conspicuously similar to another app
> 5.2.1 - Were not submitted by the legal entity that owns and is responsible for offering any services provided by the app
> 5.3.4 - Do not have the necessary licensing and permissions for all the locations where the app is used
> 
> Before we can continue with our review, please confirm that this app does not violate any of the above guidelines. You may reply to this message in Resolution Center or the App Review Information section in iTunes Connect to verify this app’s compliance. 
> 
> Given the tendency for apps of this type to violate the aforementioned guidelines, this review will take additional time. If at any time we discover that this app is in violation of these guidelines, the app will be rejected and removed from the App Store, and it may result in the termination of your Apple Developer Program account.
> 
> Should you choose to resubmit this app without confirming this app’s compliance, the next submission of this app will still require a longer review time. Additionally, this app will not be eligible for an expedited review until we have received your confirmation.
> 
> Guideline 4.2 - Design - Minimum Functionality
> 
> 
> We found that the usefulness of your app is limited by the minimal amount of content or features it includes. 
> 
> Next Steps
> 
> We encourage you to review your app concept and incorporate different content and features that are in compliance with the App Store Review Guidelines.
> 
> We understand that there are no hard and fast rules to define useful or entertaining, but Apple and Apple customers expect apps to provide a really great user experience. Apps should provide valuable utility or entertainment, draw people in by offering compelling capabilities or content, or enable people to do something they couldn't do before or in a way they couldn't do it before.
